src/sentry/integrations/repositories.py

```diff
@@ -30,6 +30,7 @@ def check_file(self, repo, filepath, branch):
         filepath: file from the stacktrace (string)
         branch: commitsha or default_branch (string)
         """
+        filepath = filepath.lstrip("/")
```

Comment:
Looks good :+1: Can you add a comment about how this prevents GitLab paths from encoding the leading slash?